functests/sctp/sctp.go

```diff
@@ -163,35 +136,59 @@ var _ = Describe("sctp", func() {
 				})
 				// OCP-26760
 				It("Should connect a client pod to a server pod", func() {
-					testClientServerConnection(client.Client, runningPod.Status.PodIP,
-						activeService.Spec.Ports[0].Port, clientNode, serverPod.Name)
+					testClientServerConnection(client.Client, testNamespace, serverPod.Status.PodIP,
+						activeService.Spec.Ports[0].Port, nodes.clientNode, serverPod.Name)
 				})
 				// OCP-26763
 				It("Should connect a client pod to a server pod. Feature LatencySensitive Active", func() {
 					fg, err := client.Client.FeatureGates().Get("cluster", metav1.GetOptions{})
 					Expect(err).ToNot(HaveOccurred())
 					if fg.Spec.FeatureSet == "LatencySensitive" {
-						testClientServerConnection(client.Client, runningPod.Status.PodIP,
-							activeService.Spec.Ports[0].Port, clientNode, serverPod.Name)
+						testClientServerConnection(client.Client, testNamespace, serverPod.Status.PodIP,
+							activeService.Spec.Ports[0].Port, nodes.clientNode, serverPod.Name)
 					} else {
 						Skip("Feature LatencySensitive is not ACTIVE")
 					}
 				})
 				// OCP-26761
 				It("Should connect a client pod to a server pod via Service ClusterIP", func() {
-					testClientServerConnection(client.Client, activeService.Spec.ClusterIP,
-						activeService.Spec.Ports[0].Port, clientNode, serverPod.Name)
+					testClientServerConnection(client.Client, testNamespace, activeService.Spec.ClusterIP,
+						activeService.Spec.Ports[0].Port, nodes.clientNode, serverPod.Name)
 				})
 				// OCP-26762
 				It("Should connect a client pod to a server pod via Service NodeIP", func() {
-					testClientServerConnection(client.Client, nodes.Items[0].Status.Addresses[0].Address,
-						activeService.Spec.Ports[0].Port, clientNode, serverPod.Name)
+					testClientServerConnection(client.Client, testNamespace, nodes.nodeAddress,
+						activeService.Spec.Ports[0].Port, nodes.clientNode, serverPod.Name)
 				})
 			})
 			AfterEach(func() {
 				deleteService(client.Client)
 				namespaces.Clean("sctptest", client.Client)
 			})
+			var _ = Describe("Test Connectivity On Default Namespace", func() {
+				var serverPod *k8sv1.Pod
+				var nodes nodesInfo
+
+				execute.BeforeAll(func() {
+					checkForSctpReady(client.Client)
+					nodes = selectSctpNodes()
+				})
+				BeforeEach(func() {
```

Comment:
Same for this one can we put it under the first describe so it will run before each describe inside the first one?